tests/features/TheLow.feature

```diff
@@ -8,9 +8,21 @@ Feature: Partywithray Proof of Membership
         Given the Partywithray Proof of Membership NFT contract is deployed
         Then the name should be "partywithray - The Low" and the symbol should "LOW"
         And the supply should be 222
+        # Do we want to mint in the constructor?  Assuming we should have a "when minted" step
         And the provenance hash should be TODOProvenanceHash
         And all 222 NFTs should have the same art
         And each NFT title should be "The Low {id}/222"
+        # Title uniqueness means we can't share metadata
+        And each NFT description should be "TODODescriptionIncludeDesignerCredit"
+        And royalties should be set at 10% going to TODOBigNightENSName
+        And the ability to mint more NFTs should be frozen
+
+    Scenario: Mint on creation
+        Then the name should be "partywithray - The Low" and the symbol should "LOW"
+        And the supply should be 222
+        And all 222 NFTs should have the same art
+        And each NFT title should be "The Low {id}/222"
+            # Title uniqueness means we can't share metadata
```

Comment:
A clarifier, that depending on how we implement metadata, we maybe can share _most_ metadata for each tier. But I know what you mean generally.

Reply:
We did discuss this.  I have it updated in another branch, actually.  Please ignore it here.